scripts/spack/packages/tribol/package.py

```diff
@@ -65,33 +69,43 @@ class Tribol(CachedCMakePackage, CudaPackage, ROCmPackage):
     depends_on("mpi")
 
     # Other libraries
-    depends_on("mfem+lapack")
+    depends_on("mfem@4.6:+lapack")
     depends_on("axom@0.9:")
 
     depends_on("raja@2024.02.0:", when="+raja")
     depends_on("umpire@2024.02.0:", when="+umpire")
+    
+    depends_on("axom+raja", when="+raja")
+    depends_on("axom~raja", when="~raja")
+    depends_on("axom+umpire", when="+umpire")
+    depends_on("axom~umpire", when="~umpire")
 
-    with when("+redecomp"):
```

Comment:
Will some of this need to be undeleted when I review the redecomp + gpu PR?

Reply:
Probably not.  Some of these dependencies were just guesses, but it's clearer now since we've implemented GPU support.